Rename getNonVariableDeprecatedTypeProperties to getDeprecatedTypeProperties

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #29667 Delete redefinitions of methods in Variable already present on Tensor.
* #29665 Move most methods off Variable into torch::autograd::impl functions.
* #29653 Remove TensorImpl::is_variable, deprecate Tensor::is_variable
* #29213 Delete all trivial uses of make_variable.
* #29204 Miscellaneous follow up for code review comments
* **#29203 Rename getNonVariableDeprecatedTypeProperties to getDeprecatedTypeProperties**

There is no more Variable/Tensor distinction, so fix the misleading name.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D18353505](https://our.internmc.facebook.com/intern/diff/D18353505)

ezyang committed:
Rename getNonVariableDeprecatedTypeProperties to getDeprecatedTypeProperties

There is no more Variable/Tensor distinction, so fix the misleading name.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

kostmo commented:
## CircleCI build failures summary
As of commit d21ebef:
* 0/1 recognized as flaky
* 1/1 failures introduced in this PR



Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=d21ebef67decebe993b6b6b98092a28d6450f3a6).

---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

ezyang committed:
Update on "Rename getNonVariableDeprecatedTypeProperties to getDeprecatedTypeProperties"


There is no more Variable/Tensor distinction, so fix the misleading name.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

ezyang committed:
Update on "Rename getNonVariableDeprecatedTypeProperties to getDeprecatedTypeProperties"


There is no more Variable/Tensor distinction, so fix the misleading name.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D18353505](https://our.internmc.facebook.com/intern/diff/D18353505)

[ghstack-poisoned]

ezyang committed:
Update on "Rename getNonVariableDeprecatedTypeProperties to getDeprecatedTypeProperties"


There is no more Variable/Tensor distinction, so fix the misleading name.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D18353505](https://our.internmc.facebook.com/intern/diff/D18353505)

[ghstack-poisoned]
